neurom/apps/morph_stats.py

```diff
@@ -73,34 +82,77 @@ def _stat_name(feat_name, stat_mode):
     return '%s_%s' % (stat_mode, feat_name)
 
 
-def extract_stats(neurons, config):
+def extract_stats(neurons, config, as_frame=False):
```

Comment:
I still wonder if it is best to have the `as_frame` to decide which function to return or if I should simply provide a different function ?

Reply:
In the cli, one would only use as_frame=False, right? This will be mostly for internal use I guess, so I'd be for leaving it as it is. I'm going to update the morph-validator code to use that function now.